ManageCoursesUi.Tests/Controllers/OrganisationControllerTests.cs

```diff
@@ -524,24 +510,20 @@ public async Task AboutPost_PublishOrganisation_invalid()
             Assert.IsNotNull(viewResult);
             var organisationViewModel = viewResult.ViewData.Model as OrganisationViewModel;
 
-            var tabViewModel = organisationViewModel.TabViewModel;
-            Assert.AreEqual(currentTab, tabViewModel.CurrentTab);
-            Assert.AreEqual(organisationName, tabViewModel.OrganisationName);
-            Assert.AreEqual(ucasCode, tabViewModel.UcasCode);
+            Assert.AreEqual(organisationName, organisationViewModel.InstitutionName);
```

Comment:
so is orgname same as instname?
if so please can you change orgname variable to instname

Reply:
The OrganisationName/InsititutionName confusion comes mostly from the API project and is pre-existing. Fixing this is more involved than the entire current PR.

Please card this up instead @defong